resource-timing/resource_TAO_multi.htm

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8" />
+<title>Resource Timing TAO tests</title>
```

Comment:
web-platform-tests should test the latest. (We should also make sure to add a test to make sure that space-separated is not supported.)